lib/plugins/clicks.js

```diff
@@ -0,0 +1,54 @@
+var global = require('global')
+var elementUtils = require('../utils/element')
+var assign = require('../utils/lodash').assign
+var disposable = require('../utils/misc').disposable
+
+function defaultOnClick (event, data) {
+  return data
+}
+
+function configure () {
+  this._clickTrackingInstalled = false
+}
+
+function trackClicks (trackClicksOptions) {
+  if (this._clickTrackingInstalled) return
+
+  var instance = this
+  var options = assign({
+    element: global.document,
+    ignoreAttribute: 'td-ignore',
+    onClick: defaultOnClick
```

Comment:
PS. The track plugin takes its options nested in `storage` and `track` objects. I wonder if the click plugin should nest in a `click` object for consistency?